server/app.js

```diff
@@ -11,6 +11,12 @@ require( 'dotenv' ).load();
 var Logger = require( './components/Logger' );
 Logger.initialize();
 
+function caca( culo ) {
+  console.log( 'pedo' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube:9000/coding_rules#rule_key=javascript%3AS2228)